### ISSUE TITLE
AwlsomeLinux [fix branch] fails to Install

### ISSUE THREAD

**Issue by AwlsomeAlex** [AwlsomeAlex/AwlsomeLinux#11]
I knew something would be broken, and that appears to be the installer. As soon as I'm done investigating the x32 problem with booting, I'll check on this.

**EDIT 1:** Here's the problem, `extlinux: not found` but that can't be because I see it in both the /sbin directory and in the build files, maybe something happened in building stage?

**EDIT 2:** Looking back at my code, I implemeted something to deal with `extlinux` being 32-bit on a 64-bit Operating System, however the library files required for installation can't be found:
`cp: cannot stat ‘/lib/ld-linux.so.2’: No such file or directory`
`cp: cannot stat ‘/lib/i386-linux-gnu/libc.so.6’: No such file or directory`
I highly dought this is a problem at all, as I am running Ubuntu ontop the Windows Subsystem for Linux, and due to it being booted by Windows, doesn't contain x32 bit files nor syslinux bootloader files. I will further experiment in a virtual machine of Ubuntu 14.04.

**Comment by ivandavidov** [comment]
Currently the installer part requires some 32-bit libraries and if they are not present on your host machine, then the installer won't work. My current workaround is to get the depending libraries directly from the host system.

You can check where these libraries are located on your local host machine by executing the following command:

``ldd extlinux``

But before that, you need to navigate to the folder where ``extlinux`` has been extracted. This is somewhere in the Syslinux structure, I'm sure you will find it.

The output of this command will give you all external library dependencies that ``extlinux`` requires and the actual library locations on your local host machines. Then you can easily copy these libraries in your MLL structure.

**Comment by AwlsomeAlex** [comment]
@ivandavidov Yeah, I figured that. I have that in the Overlay Script, but the environment which I made this was WSL so I doubt it contained the boot files which AwlsomeLinux needed. But I'll double check, if not I'll just create a beefy Virtual Machine of Ubuntu 14.04.
